### PR TITLE
address doc issues from review

### PR DESCRIPTION
**Pull request by vera-chan**
Signed-off-by: Vera <vera.xx.c@me.com>

##### SUMMARY
To adress doc issues identified by the review
Details can be found here: https://trello.com/c/IrzZ7NBS

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ IBM z/OS CICS collection
 The **IBM® z/OS® CICS® collection**, also represented as **ibm\_zos\_cics**
 in this document, is part of the broader initiative to bring Ansible® Automation to IBM Z® through the offering **Red Hat® Ansible Certified Content for IBM Z**. The IBM z/OS CICS collection supports management of CICS resources and definitions via the CMCI REST API provided by CICS.
 
-This CICS collection can work independently from other IBM z/OS modules on Ansible to perform tasks in CICS. You can, however, use it in conjunction with other IBM z/OS collections such as the 
-[IBM z/OS core collection](https://github.com/ansible-collections/ibm_zos_core) 
-to deliver a solution that will enable you to automate tasks on z/OS.
+This CICS collection can work in conjunction with or independently from other IBM z/OS collections on Ansible such as the [IBM z/OS core collection](https://github.com/ansible-collections/ibm_zos_core). When used alone, it performs automation tasks solely in CICS. When using it together with other z/OS collections, you can achieve a more powerful solution for automating tasks on z/OS.
```
Comment (stewartfrancis) [inline README.md:7]: I'll probably try and rephrase this a bit based on the feedback I got from Demetri and Dan, but it's definitely an improvement!

File: README.md [hunk 2]
```diff
@@ -23,18 +21,9 @@ Features
 
 The IBM CICS collection includes
 [modules](https://github.com/ansible-collections/ibm_zos_cics/tree/master/plugins/modules/),
-[sample playbooks](https://github.com/ansible-collections/ibm_zos_cics/tree/master/playbooks/),
+[sample playbooks](https://github.com/IBM/z_ansible_collections_samples),
```
Comment (stewartfrancis) [inline README.md:24]: We might mention that it doesn't actually "include" the sample playbooks any longer, and we could link to where the cics samples are in the sample repo, but we can update that later once it's finalized.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,10 +1,4 @@
-# How to contribute
-
-Thank you for contributing to this project.
-
-We welcome bug reports and discussions about new function in the issue tracker, and we also welcome proposed new features or bug fixes via pull requests.
-
-You should read these guidelines to help you contribute.
+# Developer guide
```
Comment (stewartfrancis) [inline CONTRIBUTING.md:1]: I think I will rename this file to dev guide at some point, based on a conversation with Demetri

**Review by stewartfrancis — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (vera-chan) [reply CONTRIBUTING.md:1]: Great, was thinking about that too! I'll update the link after it's renamed. @stewartfrancis

File: README.md (hunk 2 above)
Comment (vera-chan) [reply README.md:24]: @stewartfrancis Sounds good to me, as long as the README allows that customization~

File: README.md (hunk 1 above)
Comment (vera-chan) [reply README.md:7]: @stewartfrancis Haha sure, when making changes, pls be noted that similar wording is present in ansible_content.rst and  requirements_managed.rst, in addition to README.md.

**Pull request merged**